### ISSUE TITLE
Saving blobs failed (iOS 13)

### ISSUE THREAD

**Issue by hkunkun** [airsquared/blobsaver#107]
**Describe the bug**
Error when I click "GO" on saving blobs, it tells "Saving blobs failed". I've filled all the needed information.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS 10.14.6
 - App version v2.3.2

**Log(s)**
```
Version: 51a1c1745e58a2f26cb792615efa90d169618e4e - 336
[TSSC] manually specified generator "0x1111111111111111"
[TSSC] manually specified ECID to use, parsed "1C38290450202E" to dec:7943048165072942 hex:1c38290450202e
[TSSC] opening firmware.json
[DOWN] downloading file https://api.ipsw.me/v2.1/firmwares.json/condensed
[JSON] counting elements
[JSON] parsing elements
[TSSC] got firmware URL for iOS 13.0 build 17A577
[TSSC] opening Buildmanifest for iPhone10,3_13.0_17A577
[LFZP] downloading BuildManifest.plist from http://updates-http.cdn-apple.com/2019FallFCS/fullrestores/061-08721/B905C96C-C875-11E9-8C18-F4CC329DFA63/iPhone10,3,iPhone10,6_13.0_17A577_Restore.ipsw

PKKK?
��[Error] [LFZP] failed to open URL
[Error] [TSSC] ERROR: could not get BuildManifest for firmware URL http://updates-http.cdn-apple.com/2019FallFCS/fullrestores/061-08721/B905C96C-C875-11E9-8C18-F4CC329DFA63/iPhone10,3,iPhone10,6_13.0_17A577_Restore.ipsw

[Error] [TSSC] checking TSS status failed!
```

**Comment by airsquared** [comment]
Thanks for the bug report, I've found out the issue and I'm working on a fix.

**Comment by airsquared** [comment]
This issue has been fixed in v2.4.0.
